src/Tribe/Template.php

```diff
@@ -269,8 +278,8 @@ public function merge_context( $context = [], $file = null, $name = null ) {
 			$context = [];
 		}
 
-		// Applies local context on top of Global one
-		$context = wp_parse_args( (array) $context, $this->global );
+		// Applies new local context on top of Global + Previous local
+		$context = wp_parse_args( (array) $context, $this->get_values() );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: A cast statement must not be followed by a space

<sub>hash: 1ac4297ecddbd752d0d47a3c5ca2fb14</sub>